packages/common/src/hardhat/tasks/xchainSetup.ts

```diff
@@ -292,6 +309,13 @@ task("setup-l1-optimism-xchain", "Configures L1 cross chain smart contracts for
   }
 );
 
+task("setup-l1-blast-xchain", "Configures L1 cross chain smart contracts for Blast bridge").setAction(async function (
```

Comment:
Don't have to do this in this PR, but maybe it makes sense to start making this stuff as generic as possible.

In this case, I think having a generic task for OP Stack chains that just takes a chainId arg. Something like `yarn hardhat setup-l1-op-xchain --chainId <chainId>`.

Thoughts?

Reply:
agree, should generalize when adding new OS stack chains